### PR TITLE
feat/js-basics: add exercise dark-mode

### PR DESCRIPTION
**Pull request by F-Kirchhoff**
add a new exercise for JS Basics instead of Quiz App Challenge
https://codesandbox.io/s/github/neuefische/web-exercises/tree/feat/js-basics-dark-mode/sessions/js-basics/dark-mode?file=/js/index.js

### CODE REVIEW

File: sessions/js-basics/dark-mode/js/index.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+/*
+Add some interactivity to this website.
+ - if the body has the class "dark", the page is displayed in dark mode
+ - query all 3 buttons with the 'data-js' attribute
+ - add an event listener to each button:
+    - "switch to dark mode" button always adds the class "dark" to the body
+    - "switch to light mode" button always remvoes the class "dark" from the body
+    - "toggle mode" button toggles the class "dark" on the body
+*/
```
Comment (stefankorder) [inline sessions/js-basics/dark-mode/js/index.js:11]: This part should be written inside a readme file. You could take a look at the readme files of the objects and arrays challenges #65.

**Review by stefankorder — CHANGES_REQUESTED** [review]
Great challenge for our students. We have agreed to create a readme file for each challenge, check out my comment. My solution can be found under https://codesandbox.io/s/js-basics-dark-mode-challenge-solution-sdb792?file=/js/index.js.

File: sessions/js-basics/dark-mode/js/index.js (hunk 1 above)
Comment (PReichetanz) [reply sessions/js-basics/dark-mode/js/index.js:11]: I am going to commit a readme file in a couple of minutes.

**Review by stefankorder — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**